src/SliderButton.js

```diff
@@ -1,74 +1,83 @@
 import React, { Component, PropTypes } from 'react';
+import { Button } from 'devui';
 
 export default class SliderButton extends Component {
   static propTypes = {
     theme: PropTypes.object,
-    replaySpeed: PropTypes.string,
     type: PropTypes.string,
+    disabled: PropTypes.bool,
     onClick: PropTypes.func
   }
 
-  iconStyle(theme) {
+  shouldComponentUpdate(nextProps) {
+    return this.props.type !== nextProps.type || this.props.disabled !== nextProps.disabled;
```

Comment:
I missed that part, thanks! Even though it shouldn't be an issue here as we change the label together with this prop, it could be in future. We better use `PureComponent` here as well.